package.json

```diff
@@ -36,6 +36,5 @@
   },
   "mateAcademy": {
     "projectType": "layout"
-  },
-  "dependencies": {}
+  }
```

Comment:
when you add and commit files don't do 'git commit -am 'some message'
try 1) git add readme.md src
2) git commit -m 'some message'